python/lsst/pipe/tasks/calibrate.py

```diff
@@ -254,6 +255,10 @@ class CalibrateConfig(pipeBase.PipelineTaskConfig, pipelineConnections=Calibrate
         target=SingleFrameMeasurementTask,
         doc="Measure sources"
     )
+    setPrimaryFlags = pexConfig.ConfigurableField(
+        target=SetPrimaryFlagsTask,
+        doc="Set flags for primary source classification in single frame processing"
```

Comment:
Please very briefly define what a primary source means here

Reply:
Updated to this: `Set flags for primary source classification in single frame processing. Returns True if sources are not sky sources and have no children.`